include/odp/api/spec/packet.h

```diff
@@ -2210,6 +2261,19 @@ void odp_packet_vector_print(odp_packet_vector_t pktv);
  */
 uint64_t odp_packet_vector_to_u64(odp_packet_vector_t hdl);
 
+/**
+ * Check reassembly status of the packet
+ *
+ * When reassembly is enabled, application must check reassembly status of the
+ * packet and retrieve reassembly result from the packet before performing any
+ * packet operation.
```

Comment:
It is not clear what "reassembly result" refers to now after the recent renaming. But I do not think this sentence is even needed because now reassembly config describes the nature of an incomplete reassembly result. How about removing this sentence here and adding the following in the description of ODP_PACKET_REASS_INCOMPLETE in odp_packet_reass_status_t:

"The packet does not contain valid packet data and cannot be used in normal packet operations."

Reply:
Agreed.